crates/aptos/src/move_tool/mod.rs

```diff
@@ -40,13 +52,100 @@ impl MoveTool {
     pub async fn execute(self) -> CliResult {
         match self {
             MoveTool::Compile(tool) => tool.execute_serialized().await,
+            MoveTool::Init(tool) => tool.execute_serialized_success().await,
             MoveTool::Publish(tool) => tool.execute_serialized().await,
             MoveTool::Run(tool) => tool.execute_serialized().await,
             MoveTool::Test(tool) => tool.execute_serialized().await,
         }
     }
 }
 
+/// Creates a new Move package at the given location
+#[derive(Parser)]
+pub struct InitPackage {
+    /// Name of the new move package
+    #[clap(long)]
+    name: String,
```

Comment:
With the comment about name/package-dir below, maybe make `--name` positional argument? 
It would be just
```
aptos move init my-project
```

Reply:
I'm trying to stay away from positional arguments in general, so that it's more verbose and clear for the typing between different commands.  That's just my opinion rather than having different positional arguments